docs/pages/blog/lab-pickers-to-mui-x.md

```diff
@@ -0,0 +1,63 @@
+---
+title: The date pickers are now part of the Advanced components
+description: XXXX
+date: 2022-04-03T00:00:00.000Z
+authors: ['flaviendelangle']
+tags: ['MUI X', 'News']
+---
+
+We're excited to announce that, after more than 18 month in the lab, the date picker components are now part of the X packages.
+
+## What is MUI X ?
+
+Since its creation, MUI have always kept at heart to provide great components for free.
+However, some _Advanced components_ such as date pickers require several full-time developers and can not be sustainable relying only on open-source income.
+To be able to keep improving the library and add _Advanced components_ without compromise on code quality, we decided in 2020 to launch MUI X, a new set of components available under two licenses:
+
+- MUI X: the MIT licensed, free to use version
+- MUI X Pro: the commercially licensed version with the most advanced features
+
+The Advanced component launched by our team have been the Data Grid, with the `@mui/x-data-grid` and `@mui/x-data-grid-pro` packages.
+More than one year later, this new source of income has allowed our company to hire several new developers and to increase the amount of work achieved, even on the open-source team.
```

Comment:
Is it really necessary to explain the full backstory of MUI X before getting to the main story? I think two or three sentences would be plenty.

```suggestion
MUI X is a collection of advanced components built for complex use cases.

As opposed to our Core library, which leans on the open-source community for support, MUI X components require several full-time developers dedicated to engineering and ongoing maintenance.

MUI X components are available under two licenses:

- MUI X—MIT license; free to use
- MUI X Pro—commercial license; includes the most advanced features
```

Reply:
> Is it really necessary to explain the full backstory of MUI X before getting to the main story? I think two or three sentences would be plenty.

In your new phrasing, we just loose the information that the Date Picker is not the 1st attempt of Advanced components and that for the Data Grid it was a success in term of feature.
And that financially it also works, which is good for the user because we re-invest a good chunk of these incomes in open source development time. 

My whole blog post is here to explain that the migration is worth it for all users.